### ISSUE TITLE
Formdata parsed incorrectly

### ISSUE THREAD

**Issue by SharzyL** [SharzyL/pastebin-worker#7]
I uploaded the following paste via the frontend page:
```
print('Error 2\n', omg, '\n', theta, '\n', expse3mat, '\n', explogexpse3mat)
print('diff = ', np.max(np.abs(expse3mat - explogexpse3mat)))
break
```
When I fetched the paste, It becomes
```
print('Error 2\n', omg, '\n', theta, '\n', expse3mat, '\n', explogexpse3mat)
print('diff = ', np.max(np.abs(expse3mat - explogexpse3mat)))
brea
```

But if I upload the same text via curl, the fetched paste is exactly what I want. I suppose this is due to a buggy implementation of formdata parsing.

**Comment by SharzyL** [comment]
I will try to fix this in recent days. Before it is fixed, using cURL or CLI client is suggested.

**Comment by SharzyL** [comment]
Fixed in f5020a9bba0efff926230c4cd346675e6c2deeb4
